elasticjob-error-handler/elasticjob-error-handler-type/elasticjob-error-handler-email/src/main/java/org/apache/shardingsphere/elasticjob/error/handler/email/EmailConfiguration.java

```diff
@@ -50,4 +52,27 @@
     private String bcc;
     
     private boolean debug;
+    
+    /**
+     * Get email config.
+     *
+     * @param props props
+     * @return email config.
+     */
+    public static EmailConfiguration getByProps(final Properties props) {
+        EmailConfiguration configuration = new EmailConfiguration();
+        configuration.setHost(props.getProperty("email.host"));
+        configuration.setPort(Integer.parseInt(props.getOrDefault("email.port", "25").toString()));
```

Comment:
There are many SMTP servers required secure connection. I think it is better to let users declare port explicitly.